nixos/modules/services/misc/gitit.nix

```diff
@@ -587,6 +623,14 @@ video/x-ms-wmx  wmx
     pdf-export: ${toYesNo cfg.pdfExport}
     pandoc-user-data: ${toString cfg.pandocUserData}
     xss-sanitize: ${toYesNo cfg.xssSanitize}
+
+    [Github]
+    oauthclientid: ${cfg.oauthClientId}
```

Comment:
What happens when these variables are set to `null`? Isn't the result an evaluation error?